sites/platform/src/create-apps/_index.md

```diff
@@ -146,7 +150,9 @@ So a request for `styles.css` that accepts a gzipped file (according to the requ
 automatically returns a `styles.css.gz` file if it exists.
 This approach supports any file type and offers some CPU optimization, especially if the cache lifetime is short.
 
-## Comprehensive example
+## Comprehensive example and PHP specificities
+
+Unlike other runtimes most PHP applications do not have a start command. There is a daemon running configured to work automatically with the web server. More often than not there will be a single entry-point a "front-controller". In the case of PHP the "passthru" property is a string with the location of the front-controller rather than a boolean.
```

Comment:
```suggestion
## Comprehensive example

{{< note title="PHP specifics" theme="info" >}}

Unlike other runtimes most PHP applications do not have a start command. There is a daemon running configured to work automatically with the web server. More often than not there will be a single entry-point a "front-controller". In the case of PHP the `passthru` property is a string with the location of the front-controller rather than a boolean.

{{< /note >}}
```

Reply:
Reverts header title in favor of note to preserve link check